lib/java/com/google/android/material/card/res/values/attrs.xml

```diff
@@ -32,6 +32,10 @@
     <attr name="checkedIcon"/>
     <!-- Tint color for the checked icon. -->
     <attr name="checkedIconTint"/>
+    <!-- Check icon size -->
+    <attr name="checkedIconSize" format="dimension"/>
+    <!-- Check icon margin -->
+    <attr name="checkedIconMargin" format="dimension"/>
```

Comment:
add "for Checkable Cards" to the comments

Reply:
Done